docs/INSTALLATION.md

```diff
@@ -0,0 +1,106 @@
+## Prerequisites
+
+The Fortran language track requires that you have the following software
+installed on your system:
+* a modern Fortran compiler
+* the CMake cross-platform build system
+
+### Prerequisite: A Modern Fortran Compiler
+
+This language track requires a compiler with [Fortran 2003](https://en.wikipedia.org/wiki/Fortran#Fortran_2003)
+support. All major compilers released in the last few years should
+be compatible, so as long as you are on a fairly recent version you should be fine.
+
+The following will describe installation of [GNU Fortran](https://gcc.gnu.org/fortran/) or GFortran. Other fortran compilers are listed [here](https://en.wikipedia.org/wiki/List_of_compilers#Fortran_compilers). Intel Fortran is a popular proprietary choice for high performance applications. Most exercises will work with Intel Fortran, but are only tested with GNU Fortran so your mileage may vary.
+
+
+### Prerequisite: CMake
+
+CMake is an open source cross-platform build system that generates build
+scripts for your native build system (`make`, Visual Studio, Xcode, etc.).
+Exercism's Fortran track uses CMake to give you a ready-made build that:
+
+* compiles the tests
+* compiles your solution
+* links the test executable
+* automatically runs the tests as part of every build
+* fails the build if the any tests fail
+
+Using CMake allows exercism to provide a cross-platform build script that
+can generate project files for integrated development environments like
+Visual Studio and Xcode.  This allows you to focus on the problem and
+not worry about setting up a build for each exercise.
+
+Getting a portable build isn't easy and requires access to many kinds of
+systems.  If you encounter any problems with the supplied CMake recipe,
+please [report the issue](https://github.com/exercism/fortran/issues) so we can
+improve the CMake support.
+
+[CMake 2.8.11 or later](http://www.cmake.org/) is required to use the provided build recipe.
+
+
+#### Linux
+
+Ubuntu 16.04 and later have compatible compilers in the package manager, so
+installing the necessary compiler can be done with
+
+```bash
+sudo apt-get install gfortran cmake
+```
+
+For other distributions, you should be able to acquire the compiler through your
+package manager.
+
+#### MacOS
+
+MacOS users can install GCC with [Homebrew](http://brew.sh/) via
+
+```bash
+brew install gfortran cmake
+```
+
+#### Windows
+
+Windows users can get GNU Fortran through [MingW](http://www.mingw.org/). The easiest way is to first install [chocolatey](https://chocolatey.org).
```

Comment:
Perhaps add a note that they can also use the [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) instead and then follow the Linux instructions, since it's also a popular choice and less hassle than MingW & Chocolatey.

Reply:
I agree that linux subsystem can be useful for many things, but usually a fortran programmer on Windows would want to learn how to build native windows executable that run with other windows applications, maybe called from Python or C++.  If a person is tech-savy  enough to understand the Windows Subsystem for Linux I would expect that they can also simply follow the linux instructions. My experience with scientific computing people is that they do NOT understand much about software and platforms and mainly want to get stuff working, ie. software is a means to an end, but not their main interest.

Reply:
It may not be scientific folks that are wanting to learn about the language though.  Adding a comment on installation via Windows Linux Subsystem may point someone to an installation that they would prefer.

Reply:
I am OK with adding Linux Subsystem, but I cant test it as I am on a Windows 7 machine which does not have it. 
Lets add it when someone has been able to test it and ensure that it runs correctly, which is should as it is a simple linux executable which _should_ run in Linux Subsystem, but I have a seen many things that _should_ work, but did not. :-)

Reply:
That's how I ran the main test file from the first PR, works fine by just following the linux instructions.

Reply:
Looks like it is good to add in.